WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewController.swift

```diff
@@ -95,7 +95,11 @@ private extension AddAttributeOptionsViewController {
 
     func createUpdateIndicatorButton() -> UIBarButtonItem {
         let indicator = UIActivityIndicatorView(style: .medium)
-        indicator.color = .primaryButtonTitle
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.largeTitles) {
+            indicator.color = .accent
+        } else {
+            indicator.color = .primaryButtonTitle
+        }
```

Comment:
@pmusolino I think you may need to apply the same, since I remember seeing shipping labels using the same pattern.